.gitignore

```diff
@@ -76,3 +76,8 @@ venv/**
 
 #MacOS system files
 **/.DS_Store
+
+private
```

Comment:
please revert these changes

Reply:
Please explain why? Is this causing some problems for anyone? It is convenient for me

Reply:
if you want to ignore some specific files for your usage you should use the .git/info/exclude file.

Reply:
Thanks for the explanation, I didn't know that. I will revert these changes as requested :)